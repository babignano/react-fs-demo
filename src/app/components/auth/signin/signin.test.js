import React from 'react';
import SignIn from './signin.js';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store'
import { Provider } from 'react-redux';

const mockStore = configureStore([])

describe('SignIn', () => {
    test('should match snapshot', () => {
        const component = renderer.create(
            <Provider store={mockStore()}>
                <SignIn />
            </Provider>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
