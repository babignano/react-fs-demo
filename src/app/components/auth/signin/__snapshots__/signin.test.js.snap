// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn should match snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <label>
    Email:
    <input
      name="email"
      onChange={[Function]}
      type="email"
      value=""
    />
  </label>
  <label>
    Password:
    <input
      name="password"
      onChange={[Function]}
      type="Password"
      value=""
    />
  </label>
  <input
    type="submit"
    value="Submit"
  />
</form>
`;
